### ISSUE TITLE
Release 0.2

### ISSUE THREAD

**Issue by stefankoegl** [SoCo/socos#57]
More than a year has passed since socos 0.1, so it's probably time to think about 0.2. What do you want to get in before the release?

**Comment by petteraas** [comment]
As the changes in #55 were a bit larger than the regular pr I took the liberty to upgrade to v0.2 at the same time, I'll open a new 0.3 milestone.

**Comment by KennethNielsen** [comment]
Do you think it makes sense to skip version numbers?

**Comment by stefankoegl** [comment]
I agree -- probably not a good idea to do that, before actually doing a release.

**Comment by stefankoegl** [comment]
Does anybody know what the status of the socos release 0.2 / 0.3 is? I think it would be good to have a new release soon (see also #68).

**Comment by petteraas** [comment]
I believe I caused that mess. I haven't had time to follow this project in quite some time, so my memory may be off, but I think I only updated version in socos/__init__.py and started a new milestone 0.3.   Is it perhaps possible to reopen release 0.2 and then do it properly?
